### PR TITLE
[CDS-899] Implementation of pagination for static

### PR DESCRIPTION
**Pull request by yagin94**
# Merge request description template

## Description

**1) Root cause**

Implementation of pagination for static

**2) Changes**

https://comfortdelgrotaxi.atlassian.net/browse/CDS-899

**3) Impact**

## Check list

- [x] 1. Did you start and verify your changes?
- [x] 2. Did you run build to make sure that your changes can be built successfully?
- [x] 3. No !important flag, inline style in css
- [x] 4. No boilerplate codes (1)
- [x] 5. No duplicate code that exist in common functions?
- [x] 6. All of defined variables should have default value, check null and undefined?
- [x] 7. Use meaningful name for variables, no suffix 1,2,... Describes reference information for easier to understand what's inside. (2)
- [x] 9. Unsubscribed all of subscribers and even listeners when you don't need them.

```
(1)
Boilerplate codes is duplicated multiple times a bunch of the same code. This should be a common function to reuse through your code or you can use generic class / methods or design pattern to avoid the Boilerplate codes.
```

```
(2)
Follow theo coding convention
NO acronym variable name:
WRONG: el, elm
RIGHT: element
Your code can be read as a sentence
```

### CODE REVIEW

File: packages/static/src/pagination/styles/pagination.module.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  font-weight: 600;
+}
+
+.paginationRowsCounting {
+  font-weight: 600;
+  display: flex;
+  white-space: nowrap;
+  margin-right: auto;
+  align-items: center;
+  gap: 10px;
```
Comment (Tai-Chi-Bui) [inline packages/static/src/pagination/styles/pagination.module.css:24]: use variable

File: packages/static/src/pagination/styles/pagination.module.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  background: var(--cdg-color-cdgBlue);
+}
+
+.paginationItemActive:hover {
+  background: var(--cdg-color-cdgBlue);
+}
+
+
+.paginationItemDisabled {
+  cursor: initial;
```
Comment (Tai-Chi-Bui) [inline packages/static/src/pagination/styles/pagination.module.css:73]: should change the color & background color of active item and non-active iteam as well

File: packages/static/src/pagination/styles/pagination.module.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+.paginationItemCounting {
+  font-weight: 600;
+}
+
+.paginationRowsCounting {
+  font-weight: 600;
+  display: flex;
+  white-space: nowrap;
+  margin-right: auto;
```
Comment (Tai-Chi-Bui) [inline packages/static/src/pagination/styles/pagination.module.css:22]: why would you need margin-right when you use flex?

**Review by Tai-Chi-Bui — CHANGES_REQUESTED** [review]

File: packages/static/src/pagination/styles/pagination.module.css (hunk 3 above)
Comment (yagin94) [reply packages/static/src/pagination/styles/pagination.module.css:22]: display flex for horizontal element and margin-right: auto for create space between rowCounting component and pagin component

File: packages/static/src/pagination/styles/pagination.module.css (hunk 2 above)
Comment (yagin94) [reply packages/static/src/pagination/styles/pagination.module.css:73]: cursor initial for next-prev button

**Review by Tai-Chi-Bui — APPROVED** [review]

**Pull request merged**